src/color/Box.vue

```diff
@@ -3,7 +3,12 @@
     <span class="name">
       {{ name }}
     </span>
-    <input v-model="modelColor" class="value" />
+    <input
+      v-model="modelColor"
+      class="value"
+      @focus="handleFocus(true)"
```

Comment:
@anasvakyathodi i think better to return the original html event here

Reply:
return focus, blur separately

Reply:
got it

Reply:
done ✅